src/tabset/tabset.ts

```diff
@@ -19,6 +19,16 @@ let nextId = 0;
  */
 @Directive({selector: 'template[ngbTabTitle]'})
 export class NgbTabTitle {
+  /**
+   * CSS classes to be added to the tab
+   * The CSS classes are updated as follows, depending on the type of the expression evaluation:
+   * - `string` - the CSS classes listed in the string (space delimited) are added,
+   * - `Array` - the CSS classes declared as Array elements are added,
+   * - `Object` - keys are CSS classes that get added when the expression given in the value
+   *              evaluates to a truthy value, otherwise they are removed.
+   */
+  @Input() ngbTabTitleClass: string | string[] | Set<string>| {[klass: string]: any};
```

Comment:
Apparently trying to name it `[ngClass]` causes some kind of conflict or something.  The unit test gives me an error "Cannot read property 'add' of undefined" when setting up the html.  I'll just use the requested `titleClass`.